CHANGES.rst

```diff
@@ -1,6 +1,25 @@
 Changes
 =======
 
+Version 0.5.0 (2019-04-16)
+--------------------------
+
+- Introduces new workflow statuses: ``deleted``, ``stopped``, ``queued``.
+- Adds new field to store workflow stopping time, ``run_stopped_at``.
+- Moves workflow input parameters to its own column, ``input_parameters``.
+- Renames ``get_parameters`` to ``get_input_parameters`` because there are now
+  two types of parameters, ``input_parameters`` and ``operational_parameters``.
+- Adds a new enumeration for possible job statuses ``JobStatus``.
+- Adds new field in the ``Job`` model, ``backend_job_id``, to identify jobs
+  in the underlying compute backend.
+- Introduces new fields for interactive sessions: ``interactive_session``,
+  ``interactive_session_name`` and ``interactive_session_type``. Note that with
+  current design only one interacitve session per workflow is supported.
```

Comment:
- interactive type

Reply:
One more thing:  perhaps add minor hints at the meaning, as you nicely did for:

- Useful to identify jobs in the underlying compute backend

so that one could have:

- Moves workflow input parameters to its own column to separate them from operational options. Adapts getters accordingly. (``Workflow.input_parameters``)

so that there is both an easily readable sentence and a reference to the DB column at the end.  (In case we want to standardise that way for easier lookup...)